examples/app.py

```diff
@@ -91,7 +91,7 @@ def load():
 
     # Record 1 - Live record
     with db.session.begin_nested():
-        rec_uuid = uuid.uuid4()
+        rec_uuid = '9107e6ef-fea4-4971-ae2f-934d2fdcaa34'
```

Comment:
shall we choose something artificial like: `deadbeef-1234-5678-ba11-b100dc0ffee5'